app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_objects = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+
+    for person in people:
+        person_object = Person.people[person["name"]]
+
+        if person.get("wife") is not None:
+            wife = Person.people[person["wife"]]
+            person_object.wife = wife
+        elif person.get("husband") is not None:
+            husband = Person.people[person["husband"]]
+            person_object.husband = husband
```

Comment:
The code assumes that the spouse (either 'wife' or 'husband') will always exist in the `Person.people` dictionary. If a spouse is not included in the initial list of people, this will raise a KeyError. Consider adding a check to ensure the spouse exists in `Person.people` before attempting to access it.